### DISCUSSION TITLE
Dose this package support float number with out limit such as BigInteger

### DISCUSSION THREAD

**Discussion by TruePluto** [mathnet/mathnet-numerics#823] [Q&A] [answered]
I want to processor some number very.... big.
The standard dotNet just support the integer via BigInteger.
But how to processor float? thank you.

**Comment by cdrnet** [comment, marked as answer]
If you reference MathNet.Numerics.FSharp (works in C# as well) in addition to MathNet.Numerics, you'll find a type called `BigRational`, which is essentially representing arbitrary precision rational numbers (built using two BigIntegers). The explicit semantics of arbitrary precision rational numbers are a bit different to using floats, but they can likely represent the structure you're looking for.

**Comment by TruePluto** [comment]
thank you, but Its just store two biginteger and without computer, maybe its not mature.
but thanks you for your attention.

**Comment by cdrnet** [comment]
It is used in production, so seems to be mature enough. But your use case my differ. Thanks for the update.

**Comment by shtse8** [comment]
Any C# version of BigRational?

**Comment by cdrnet** [comment]
BigRational should work well from C# - have you run into a problem or a case that doesn't work well?

**Comment by shtse8** [comment]
I can't find the BigRational class in C# MathNet. :(
